pkg/apis/k0s/v1beta1/network.go

```diff
@@ -114,6 +114,15 @@ func (n *Network) Validate() []error {
 			errors = append(errors, field.Invalid(field.NewPath("podCIDR"), n.PodCIDR, "podCIDR and serviceCIDR must be both IPv4 or IPv6"))
 		}
 
+		// Validate IPv6 ServiceCIDR prefix length per Kubernetes requirements (<= /108).
+		// https://github.com/kubernetes/kubernetes/blob/v1.34.2/cmd/kubeadm/app/apis/kubeadm/validation/validation.go#L461-L478
```

Comment:
Good to provide an upstream reference! However, I think this one is more appropriate:

```suggestion
		// https://github.com/kubernetes/kubernetes/blob/v1.34.3/cmd/kube-apiserver/app/options/validation.go#L86-L96
```